uktrade/data-hub-frontend
# Feature/maxemail campaign

## Description of change
This piece of work integrates Maxemail email campaigns into the Activity Feed of Data Hub. 

More specifically, Data Hub company contacts that have been sent an email as part of a Maxemail campaign are now grouped into their respective campaigns within the Activity feed. This allows users of Data Hub to view who were sent Maxemail emails as part of a particular campaign.

Each Maxemail campaign displays the following:
* The title of the email
* The date of the email
* The sender's name
* The sender's email address
* The content/summary
* A list of Data Hub company contacts (the recipients) and are linked to the contacts page.

This work is hidden behind a feature flag called `activity-feed-maxemail-campaign` which is currently enabled both in `dev` and `staging` and disabled in `prod`.

## Test instructions
Bring up the FE locally and point to the dev API on PaaS, open a browser and go to:
_/companies/6df487c5-7c75-4672-8907-f74b49e6c635/activity_

Now apply the `All external activity` filter

Here you will see the following...

<img width="971" alt="maxemail-campaign" src="https://user-images.githubusercontent.com/964268/102713535-e2bb1e80-42c0-11eb-98fb-ceee0e16fe6d.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
